src/app/requests/RefreshRequest.ts

```diff
@@ -0,0 +1,35 @@
+import { CommandContext } from 'slash-create';
+import { Request } from './Request';
+import { MessageReactionRequest } from '../types/discord/MessageReactionRequest';
+import { Activities } from '../constants/activities';
+import { ButtonInteraction, Message } from 'discord.js';
+import DiscordUtils from '../utils/DiscordUtils';
+
+export class RefreshRequest extends Request {
+    bountyId: string;
+
+    commandContext: CommandContext;
+    message: Message;
+    buttonInteraction: ButtonInteraction;
+
+    constructor(args: {
+        commandContext: CommandContext,
+        messageReactionRequest: MessageReactionRequest,
+        buttonInteraction: ButtonInteraction,
+    }) {
+        if (args.commandContext) {
```

Comment:
This code is unreachable I think. Should never be a command context for refresh

Reply:
Right. Can remove it

Reply:
fixed